### ISSUE TITLE
Reverse proxy websocket?

### ISSUE THREAD

**Issue by MaxWeichers** [fxclient/FXclient#3]
wss://Territorial.io/socket is blocked at my school, you can set up a reverse proxy with nginx or cloudflare

**Comment by peshomir** [comment]
I assume you mean `wss://territorial.io/s52/`, don't know of `/socket` being used in recent versions at least. Are the actual lobby and game servers (subdomains of territorial.io) blocked too?

**Comment by MaxWeichers** [comment]
yes all subdomains, You could setup a reverse proxy with cloudflare workers, it is free 100k requests per day

**Comment by mohsenemx** [comment]
I will take a look on that
